docs/src/pages/landing-page/Index.vue

```diff
@@ -196,4 +304,22 @@ export default defineComponent({
 .support {
   height: 100vh;
 }
+
+.my-grid {
+  font-size: .5rem;
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
+  grid-column-gap: 1rem;
+  grid-row-gap: 2rem;
+  align-content: center;
+}
+
+.final-words-section {
+  // TODO: reduce image height (from top), and remove this negative margin
```

Comment:
Is this something Nick should take care?

Reply:
Yeah, I needed this PR done with so I can discuss the changes with Nick.